stock_secondary_unit/models/stock_move.py

```diff
@@ -12,16 +12,36 @@ class StockMove(models.Model):
     }
 
     product_uom_qty = fields.Float(
-        store=True, readonly=False, compute="_compute_product_uom_qty", copy=True
+        store=True,
+        readonly=False,
+        compute="_compute_product_uom_qty",
+        copy=True,
+        precompute=True,
     )
 
     @api.depends("secondary_uom_qty", "secondary_uom_id")
     def _compute_product_uom_qty(self):
         self._compute_helper_target_field_qty()
 
-    @api.onchange("product_uom")
-    def onchange_product_uom_for_secondary(self):
-        self._onchange_helper_product_uom_for_secondary()
+    @api.depends("product_id")
+    def _compute_product_uom(self):
```

Comment:
@rov-adhoc Could you try to remove completely the compute as it should be computed by the mixin.

See: https://github.com/OCA/product-attribute/pull/1560/files#diff-4111726e152eb08dc8a785703aa8b37a47652a6d7abb0d13ab97b41da579c9eaR18

Reply:
@rousseldenis You're right!
Thanks